runtimes/elixir-1.14/server/test/test_helper.exs

```diff
@@ -0,0 +1 @@
+ExUnit.start()
```

Comment:
Is this unit test for HTTP server? If so, I think we can remove this. Our tests system should have good enough coverage.